bench/src/main/scala/org/http4s/bench/HeadersBench.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015 http4s.org
```

Comment:
2022? :)

Reply:
- https://github.com/http4s/http4s/pull/5525

Reply:
But this is a new file, so it shouldn't say 2015, afaict? Ember files are mostly "Copyright 2019 http4s.org", for example. Oh well, not terribly important.

Reply:
- https://github.com/typelevel/sbt-typelevel/issues/353

Reply:
Sorry to be honest I just copied over the copyright from another file, didn't think about it much!

Reply:
Everything defaults to `startYear` for the project.  We _could_ parse git, but this would be prohibitively slow on a project of this size.  If you delete the header and run `headerCreateAll`, it should create one that's consistent with the project configuration.